src/middlewared/middlewared/plugins/audit/schema/smb.py

```diff
@@ -65,6 +66,33 @@ class AuditSetattrType(AuditEnum):
 """
 
 
+AUDIT_EVENT_DATA_SMB_AUTHENTICATION = Dict(
+    str(AuditEventParam.EVENT_DATA),
+    Str('logonId'),
+    Int('logonType'),
+    Str('localAddress'),
```

Comment:
`status` appears to be present in samba/auth/auth_log.c

If it's not supposed to be present here, then please add a comment explaining why.

Also wondering about `clientPolicyAccessCheck` and `serverPolicyAccessCheck.`

Reply:
We use a slightly different function for this than what is in auth/auth_log.c. Status is remapped to `result` and `clientPolicyAccessCheck` and `serverPolicyAccessCheck` are dropped due to being AD DC specific.